src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -114,9 +114,9 @@ private void ReloadScript()
         {
             // Reset cached function
             ResetFunctionValue();
-            TraceWriter.Info(string.Format(CultureInfo.InvariantCulture, "Script for function '{0}' changed. Reloading.", Metadata.Name));
+            TraceOnPrimary(string.Format(CultureInfo.InvariantCulture, "Script for function '{0}' changed. Reloading.", Metadata.Name), TraceLevel.Info);
```

Comment:
Decided to go with the simpler approach for now and just have the logic to conditionally trace in the invoker (as opposed to another, conditional trace writer). We can revisit this if our needs change.